cosmo_tester/test_suites/bootstrap_based_tests/inplace_upgrade_test.py

```diff
@@ -54,64 +53,18 @@ def test_inplace_upgrade(cfy,
                               module_tmpdir, logger)
     for hello_world in hellos:
         hello_world.upload_and_verify_install()
-    create_snapshot(manager, snapshot_name, attributes, logger)
```

Comment:
can we now remove the `create_snapshot` and friends (restore etc), if we can also replace other call sites?

Reply:
I'd rather not - these calls are making extra checks and validations that are relevant for snapshot tests, and I think we better leave those in.